src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts

```diff
@@ -84,6 +106,14 @@ export class DotHttpErrorManagerService {
         return false;
     }
 
+    private handleLicense(): boolean {
+        this.dotDialogService.alert({
+            message: this.dotMessageService.get('dot.common.http.error.403.license.message'),
+            header: this.dotMessageService.get('dot.common.http.error.403.license.header')
+        });
+        return false;
```

Comment:
because this is use here

https://github.com/dotCMS/core-web/pull/649/files/8102d6dbe450ee194068ff0104f6009e32f1cbf7#diff-52fdbb05153322a37c21963499c59746R56